src/visual.ts

```diff
@@ -1381,11 +1381,11 @@ module powerbi.extensibility.visual {
                 // Update each link related with this node
                 node.links.forEach( (link: SankeyDiagramLink) => {
                     // select link svg element by ID generated in link creation as Source-Destination
-                    d3.select(`#linkLabelPaths${(link.source.label.name || "").replace(/\W*/g,"")}-${(link.destination.label.name || "").replace(/\W*/g,"")}`).attr({
+                    d3.select(SankeyDiagram.createLink(link, true) ).attr({
```

Comment:
Change to d3.select(`#` + SankeyDiagram.createLink(link, true) ).attr({
#is required to select by id.